### ISSUE TITLE
Adjust color scheme (in dark mode?)

### ISSUE THREAD

**Issue by dlangille** [FreshPorts/freshports#330]
From https://twitter.com/AfoPTech/status/1429320873585123330

How about those colors:
```
background-color: #151515
code-text: #dfc7ff
link-color: #33cdff
```

They raise the contrast, but keep the color-scheme.

**Comment by dlangille** [comment]
@jethron would you be able to figure this one out please?

**Comment by dlangille** [comment]
Is this looking OK at https://dev.freshports.org/ and https://test.freshports.org/?

**Comment by weust1979** [comment]
dvl over at libera/#freebsd IRC asked me to comment here one how I perceive the purple description texst.

Too me it feels like too much black is coming through the purple text making it fairly hard to read.
Kinda feels like words are jumping a bit.

I've attached a screenshot to show what I see on my display.
This was captured on Windows 10 running Dark Mode using Firefox with font settings default.

![Screenshot 2021-09-08 at 17-21-20 FreshPorts -- cad PrusaSlicer Slicing application for 3D printers](https://user-images.githubusercontent.com/11320409/132539069-29b71f42-8f46-4b9a-b2cc-5f5f62b9a00f.png)

**Comment by dlangille** [comment]
Via IRC, it was the description which was hardest to read. How about white?

**Comment by weust1979** [comment]
The white is fine. Like what you can read at Broken or Ignore lines.
Not sure if that's a different font too?

It feels like the purple text has less body to each letter. If that makes sense.

**Comment by jethron** [comment]
Thanks @weust1979 , good feedback! They are different fonts, yes; the purple text uses the "monospace" placeholder which is configurable by your browser preferences - but on Windows this unfortunately tends to default to Courier New which has a lot less body, you're correct. I've got a branch over on https://a.freshports.org/cad/PrusaSlicer/ which preferences "Lucida Console" ahead of straight monospace. It's a relatively similar font (t and r look to have the biggest differences), but has more "body", and is slightly taller. 

Pretty limited when it comes to monospace fonts built into Windows, but another option is Consolas. This is pretty popular as it's the default in e.g. VS Code. But it's rather thinner than the Courier New default most Windows folk see now.

We could also look at pushing a web font to get something reasonably consistent looking across platforms, but that's going further into redesign territory than I'm qualified to take us. :)

Let me know what you think of Lucida. I've made the change in both light & dark styles at the moment, but could keep it just on the dark theme if needed.
